llpc/patch/llpcPatchDescriptorLoad.cpp

```diff
@@ -512,8 +528,23 @@ Value* PatchDescriptorLoad::LoadDescriptor(
         }
         else
         {
-            auto pDescOffset = ConstantInt::get(Type::getInt32Ty(*m_pContext), descOffset);
-            auto pDescSize   = ConstantInt::get(Type::getInt32Ty(*m_pContext), descSize, 0);
+            llvm::Value* pDescOffset = nullptr;
+            if (!m_pPipelineState->GetBuilderContext()->BuildingRelocatableElf())
+            {
+                pDescOffset = ConstantInt::get(Type::getInt32Ty(*m_pContext), descOffset);
+            }
+            else
+            {
+                IRBuilder<> builder(*m_pContext);
+                builder.SetInsertPoint(pInsertPoint);
+                SmallVector<Value*, 16> args;
+                args.push_back(ConstantInt::get(Type::getInt32Ty(*m_pContext), descSet));
+                args.push_back(ConstantInt::get(Type::getInt32Ty(*m_pContext), binding));
```

Comment:
builder.getInt32; also, prefer feeding the args directly to CreateIntrinsic instead of a SmallVector (using {arg0, arg1} syntax).